core/src/main/scala/org/apache/spark/rpc/netty/NettyStreamManager.scala

```diff
@@ -58,7 +59,7 @@ private[netty] class NettyStreamManager(rpcEnv: NettyRpcEnv)
         new File(dir, fname)
     }
 
-    if (file != null && file.isFile()) {
+    if (file != null && file.isFile) {
```

Comment:
While these changes are still correct behavior-wise, these are java methods that are defines as zero-arg methods, so including parens is a little more correct